vraptor-core/src/main/java/br/com/caelum/vraptor/util/SafeResourceBundle.java

```diff
@@ -59,6 +63,7 @@ protected Object handleGetObject(String key) {
 		try {
 			return delegate.getString(key);
 		} catch (MissingResourceException e) {
+			logger.warn("Resource missed while calling delegate", e);
```

Comment:
note: in this case makes totally sense to use logger#warn, since it is the developer's fault